MovieApp/PresentationLayer/MovieDetails/MovieHeader/CircularRating/CircularRatingView+Design.swift

```diff
@@ -0,0 +1,78 @@
+import UIKit
+
+extension CircularRatingView: ConstructViewsProtocol {
+
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    func createViews() {
+        circleLayer = CAShapeLayer()
+        ratingLayer = CAShapeLayer()
+
+        let circularPath = UIBezierPath(
+            arcCenter: CGPoint(x: radius, y: radius),
+            radius: radius,
+            startAngle: -.pi/2,
+            endAngle: 3 * .pi/2,
+            clockwise: true)
+
+        circleLayer.path = circularPath.cgPath
+        ratingLayer.path = circularPath.cgPath
+
+        layer.addSublayer(circleLayer)
+        layer.addSublayer(ratingLayer)
+
+        ratingLabel = UILabel()
+        addSubview(ratingLabel)
+    }
+
+    func styleViews() {
+        circleLayer.fillColor = UIColor.clear.cgColor
+        circleLayer.lineCap = .round
+        circleLayer.lineWidth = lineThickness
+        circleLayer.strokeColor = UIColor(named: "DarkGreen")?.cgColor
+
+        ratingLayer.fillColor = UIColor.clear.cgColor
+        ratingLayer.lineCap = .round
+        ratingLayer.lineWidth = lineThickness
+        ratingLayer.strokeEnd = 0
+        ratingLayer.strokeColor = UIColor(named: "LightGreen")?.cgColor
+
+        styleRatingLabel()
+    }
+
+    func styleRatingLabel(for string: String = "%") {
+        let numberAttributes = [NSAttributedString.Key.font: UIFont(name: "ProximaNova-Bold", size: 15),
```

Comment:
`NSAttributedString.Key.font: UIFont(name: "ProximaNova-Bold", size: 15),` in new line